test/test.js

```diff
@@ -403,6 +403,37 @@ describe("multi build", function(){
 		});
 	});
 
+	it("should pass the traceurOptions to transpile", function(done){
+		console.log('starting test');
+		rmdir(__dirname + "/es6-loose/bundle", function(error){
+			if(error) {
+				return done(error);
+			}
+
+			multiBuild({
+				config: __dirname + "/es6-loose/config.js",
+				main: "main",
+			}, {
+				quiet: true,
+				minify: false,
+				traceurOptions: {
```

Comment:
Yes, you're right. I'll make these changes. `babelOptions` should also be in the first set of options, right? I'll need to update that also.